Description: Fix a non-POSIX sh syntax error in a make rule.
Author: Fabian Greffrath <fabian@debian.org>
Last-Update: 2021-08-20

--- a/Makefile.am
+++ b/Makefile.am
@@ -47,7 +47,7 @@ SHELL=$(if $(wildcard @KLEKNEV@),@KLEKNE
 
 # SILENT BUILD BLINKENLIGHTS
 
-TSU_V_REDIR=$(if $(AM_V_at),>& /dev/null)
+TSU_V_REDIR=$(if $(AM_V_at),> /dev/null 2>&1)
 
 if COND_COLOUR
   escchar:=$(strip $(shell echo x | tr 'x' '\033'))
